### DISCUSSION TITLE
Problem with school ID and its approval.

### DISCUSSION THREAD

**Discussion by zhenias** [community/community#140931] [GitHub Education]
### Hi everyone,

Hi everyone.

I have problem.
![image](https://github.com/user-attachments/assets/cf44c386-6b0c-44cb-a591-4143c5f4aca1)

They don't answer anything in support.
The following message appears when I sent my school ID - in Polish.
I have already tried to send the application 61 times - to no avail.
I tried the paper version, electronic version, and recording (screenshot).

Thanks you.

**Comment by xander-doom** [comment]
I sent my statement of account and they accepted that. My student ID didn't have an expiration date on it so it wasn't valid.

**Reply by zhenias** [reply]
I'm studying at a technical school - I don't pay for school.

**Reply by xander-doom** [reply]
The education discount requirements state that you must "Be currently enrolled in a degree or diploma granting course of study from a recognized educational institution." Does your school fulfil that? Do you have a .edu email?

**Reply by zhenias** [reply]
I am studying - the document confirming that I am attending is my school ID. 
Regarding `.edu` e-mail, our school does not provide us with such things, only teachers have e-mail from `.edu`.

Can anything help me get the github pack discount?
I also wrote to GitHub Support about this - no response on this topic, I just have to keep trying.
